cephcsi/Dockerfile

```diff
@@ -0,0 +1,70 @@
+# ceph-csi container
+ARG SRC_DIR="/work/go/src/github.com/ceph/ceph-csi/"
+ARG GO_ARCH=amd64
+ARG BASE_IMAGE="quay.io/cybozu/ceph:16.2.7.0"
+
+FROM ${BASE_IMAGE} as builder
+
+LABEL stage="build"
+
+ARG CSI_IMAGE_NAME=quay.io/cybozu/cephcsi
+ARG CSI_IMAGE_VERSION=3.5.1
+ARG GO_ARCH
+ARG SRC_DIR
+ARG GIT_COMMIT
+ARG GOROOT=/usr/local/go
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN apt-get update && apt-get -y install \
+    git \
+    && true
```

Comment:
Why it is needed? Did you copy this from upstream?
https://github.com/ceph/ceph-csi/blob/4652b8facfa56f6619967f1466b548bd6a924f19/deploy/cephcsi/image/Dockerfile#L38

Reply:
>Did you copy this from upstream?

Exactly. But, honestly, I was not sure understanding why it is needed.
According to the following document, it is to prevent editing errors.
https://github.com/projectatomic/container-best-practices/blob/master/creating/creating_index.adoc
>Editing a line with a command that that you don’t want to change presents a risk of introducing a bug, and you also obscure the line’s history. This can be mitigated by having both the first and last commands true - they don’t do anything.

In our case, only git should be installed here for `git clone`, and it is unlikely that other programs are added (other programs may be added to the parameter list of another `apt-get install` command appearing later). So I removed it.

Reply:
Got it